src/node_crypto.cc

```diff
@@ -155,7 +155,6 @@ std::string extra_root_certs_file;  // NOLINT(runtime/string)
 X509_STORE* root_cert_store;
 
 // Just to generate static methods
-template class SSLWrap<TLSWrap>;
```

Comment:
Could it be that this was supposed to generate the vtable or something? But I agree, I don’t really see the point in having any of the methods listed here … there should be an `extern template` matching them, or they should not be there at all, right? (and I would prefer the latter.)